pages/api/auth/forgot-password.ts

```diff
@@ -29,13 +29,24 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
       },
       select: {
         name: true,
+        identityProvider: true,
       },
     });
 
     if (!maybeUser) {
       return res.status(400).json({ message: "Couldn't find an account for this email" });
     }
 
+    // if (maybeUser.identityProvider !== IdentityProvider.CAL) {
```

Comment:
The code changed here since my changes, need to look at how to modify the email template to plug in this message

Reply:
@zomars No, best to remove it and take it up properly in a follow up PR